languagetool-language-modules/pt/src/main/resources/org/languagetool/resource/pt/hunspell/spelling.txt

```diff
@@ -6511,6 +6521,8 @@ putinha
 putinhas
 putona
 putonas
+buceta
+bucetas
```

Comment:
The correct spelling for this word is with an `o`, not an `u`. I see it does not appear as a suggestion in the editor.

Reply:
Should it even appear? Tbh the policy on vulgarity isn't super clear to me, either. I can delete it, I suppose, but I feel like policing the 'proper' spelling of a very common variant of such an informal word is a bit too prescriptivist? I can delete it if that's the policy, but weird.

Reply:
I have, of course, completely neglected the erotica authors!